libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/stats/mysql/OBMySQLNoLessThan400StatsAccessor.java

```diff
@@ -41,6 +41,7 @@ public class OBMySQLNoLessThan400StatsAccessor extends OBMySQLStatsAccessor {
             + "  STATE, "
             + "  `USER_CLIENT_IP` as HOST, "
             + "  HOST as PROXY_HOST, "
+            + "  CONCAT(SVR_IP, ':', SVR_PORT) AS SVR_IP, "
```

Comment:
Does ob mysql and oracle support this field only in versions greater than 4.0?

Reply:
Yes， and only  ob versions greater than 4.0 will use svrIp as well.